core/src/components/cat-select/cat-select.tsx

```diff
@@ -577,7 +578,8 @@ export class CatSelect {
                 label={this.state.selection[0].render.label}
                 round={this.state.selection[0].render.avatar.round}
                 src={this.state.selection[0].render.avatar.src}
-                initials={''}
+                initials={this.state.selection[0].render.avatar.initials ?? ''}
+                size={this.state.selection[0].render.avatar.initials ? 'xs' : 'm'}
```

Comment:
connecting the size of the avatar to the initials does not make sense to me. Why was it implemented this way?